CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
-# dbt_iterable v0.13.1
+# dbt_iterable v0.13.2
```

Comment:
This is changing the file type of the incremental model. This needs to be a breaking change

Reply:
Additionally, this will require a full refresh for Databricks users.